pkg/velero/restore.go

```diff
@@ -0,0 +1,32 @@
+package velero
+
+import (
+	"sort"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetRestoreNamespace return the namespace mapping for the given namespace
```

Comment:
The comment can be modified to make it clear that target namespace is returned by this function. 

```
GetRestoreNamespace return the target namespace for restoring from the backup (bkpName). 
If namespace mapping was provided to the restore command then target namespace is returned. 
If namespace mapping was not provided to the restore command, source namespace (ns) will be returned. 
```

Reply:
ok.